MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
+        /// </summary>
+        public class Random
```

Comment:
@KonajuGames @dellis1972 - Do we really want this class to be nested like this?  I'm asking here one final time as we will be sort of stuck with the API once we merge.